src/System.Private.ServiceModel/tools/scripts/WCFCustomCleanup.cmd

```diff
@@ -0,0 +1,36 @@
+@if "%_echo%" neq "on" echo off
+setlocal
+
+REM
+echo Cleanup test services
+REM  Note that we always return successful code in clean up routine because clean up errors are not fatal
+
+TASKKILL /F /IM SelfHostedWCFService.exe
+REM error 128 will be returned if The process not found
+if NOT [%ERRORLEVEL%]==[0]  if NOT [%ERRORLEVEL%]==[128] (
+	echo Warning: An error occurred while killing the SelfHostedWCFService.exe.
+	)
+
+call %~dp0CleanUpHttpsPort.cmd
```

Comment:
Not blocking for review -- the reason I want to deal with making ports configurable is that we are accumulating debt in all the places that should forward those parameters, like here.